### PR TITLE
DFPDP # 199 Partner Portal - Get Assistance

### PR DESCRIPTION
**Pull request by shruthi-rapala**
# Description

This PR includes the following proposed change(s):
DFPDP # 199 Partner Portal - Get Assistance
- Updated all the callback controller methods to use GetDriverInfo method
- Updated styles on callbacks
- Added call back description Enum
- Updated backend API for call back end points

## Type of change

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

### Screenshots

Include screen captures or recordings with captions of the tests run (unit tests, postman request/response and or manual UI driven tests.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [X ] No

## Checklist:

- [ X] My code follows the [style guidelines](https://justice.gov.bc.ca/wiki/display/JAM/RCC+Style+Guide) of this project
- [ X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: partner-portal/src/UI/src/app/features/auth/enums/call-back.enum.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+export const CallStatusDescription = new Map<number | undefined, string>([
+  [0, 'Open'],
+  [1, 'Closed']
+])
```
Comment (jburditt) [inline partner-portal/src/UI/src/app/features/auth/enums/call-back.enum.ts:5]: move to a different folder. I'm not really sure what folder, there is a folder structure I try to follow that other developers follow, I'll see if I can find the link

Comment (jburditt) [reply partner-portal/src/UI/src/app/features/auth/enums/call-back.enum.ts:5]: https://itnext.io/choosing-a-highly-scalable-folder-structure-in-angular-d987de65ec7

And after skimming through it I still don't know where to put it

Comment (jburditt) [reply partner-portal/src/UI/src/app/features/auth/enums/call-back.enum.ts:5]: /shared/enums

https://github.com/bcgov/moh-pidp/blob/develop/workspace/apps/pidp/src/app/shared/enums/roles.enum.ts

Comment (shruthi-rapala) [reply partner-portal/src/UI/src/app/features/auth/enums/call-back.enum.ts:5]: @jburditt  I have added this folder to shared enums level where the enums for the identity provider exists

**Review by jburditt — APPROVED** [review]

**Pull request merged**